backend/src/tests/integration/user.spec.js

```diff
@@ -0,0 +1,43 @@
+const request = require('supertest');
+const app = require('../../src/app');
+const connection = require('../../src/database/connection');
+
+describe('User', () => {
+  beforeAll(async () => {
+    await connection.migrate.rollback();
+    await connection.migrate.latest();
+  });
+
+  afterAll(async () => {
+    await connection.destroy();
+  });
+
+  it('Create a new User', async() => {
```

Comment:
Suggestion: add a test case when the database insertion is faulty.

Reply:
Done